Dgpu 15

yhe39 opened the pull request.

sysopenci commented:
Improper Commit Message
Valid Commit Message
Improper Jira Status,
    jira status not in ['In Progress','Implemented']

sysopenci commented:
Improper Commit Message
Proper Tracked on value is not present in
    commit message,make sure Tracked-on: jira-ticket is present

sysopenci commented:
Improper Commit Message
Proper Tracked on value is not present in
    commit message,make sure Tracked-on: jira-ticket is present

yhe39 committed:
Add the special gralloc private 2 flag check

If has the GRALLOC_USAGE_PRIVATE_2, use local memory,
therwise use the system memroy.
By default use the sys memory, dGPU only changes
to use local memory to improve performance.

Tracked-On: OAM-130613
Signed-off-by: HeYue <yue.he@intel.com>

feijiang1 reviewed (approved):
LGTM

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is SUCCESS. Please check the linked Tracked-On issue/Android CI Web  for more details. For Binaries: /cactus-absp-or-local/celadon-merge/1156
